cmd/apply.go

```diff
@@ -98,8 +109,23 @@ func apply(c *cobra.Command, _ []string) error {
 	return nil
 }
 
+func priorityTypeOf(prioirty string) (spanner.PriorityType, error) {
+	switch prioirty {
+	case "high":
+		return spanner.PriorityTypeHigh, nil
+	case "medium":
+		return spanner.PriorityTypeMedium, nil
+	case "low":
+		return spanner.PriorityTypeLow, nil
+	default:
+		return 0, fmt.Errorf("%s is unsupported priority, it must be one of high, medium, or low", priority)
+	}
+
+}
+
 func init() {
 	applyCmd.PersistentFlags().StringVar(&ddlFile, flagDDLFile, "", "DDL file to be applied")
 	applyCmd.PersistentFlags().StringVar(&dmlFile, flagDMLFile, "", "DML file to be applied")
 	applyCmd.PersistentFlags().BoolVar(&partitioned, flagPartitioned, false, "Whether given DML should be executed as a Partitioned-DML or not")
+	applyCmd.PersistentFlags().StringVar(&priority, flagPriority, "high", "The priority to apply DML (optional. if not set, will apply as high priority task")
```

Comment:
Given: https://github.com/cloudspannerecosystem/wrench/pull/37/files#r644431377,

```suggestion
	applyCmd.PersistentFlags().StringVar(&priority, flagPriority, priorityHigh, "The priority to apply DML (optional. if not set, will apply as high priority task")
```